src/main/java/frc/robot/RobotContainer.java

```diff
@@ -35,7 +37,7 @@ public static RobotContainer getInstance() {
 
     private void configureDefaultCommands() {
         drive.setDefaultCommand(
-                new KeyboardDriveSim()
+                new XboxDrive(new XboxController(0))
```

Comment:
Don't create the instance the Xbox controller here. It will surely be useful for other things later